CSS/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  font-family: Helvetica;
+  font-size: 18px; 
+  line-height: 1.5;
+  padding: 50px;
+  background-color: #eeeeee;
+  text-align: center;
+}
+
+a {
+  font-family: Helvetica;
+  font-size: 30px;
+  font-weight: bold;
+  color: black;
+  padding: 20px;
+  text-decoration: none;
+}
+
+p {
+  font-family: Helvetica; 
+}
+
+img {
+  width: 960px;
+  margin: 20px 60px 0 60px;
+
+}
+
+img.aboutme {
```

Comment:
Combining tags and classes is not necessary and drives up the specificity without need.  Please read this article on specificity (https://css-tricks.com/specifics-on-css-specificity/) and BEM (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/).  BEM is the most popular of the modular strategies to handle styling with classes (which is the industry best practice).  It is used by Google, AirBnB, etc, and is my go to way of organizing CSS. 🔨